### PR TITLE
fix black format in ci

### PR DESCRIPTION
**Pull request by qiaocci**

### CODE REVIEW

File: iredis/client.py [hunk 1]
```diff
@@ -80,7 +80,9 @@ def get_server_info(self):
         # safe to decode Redis's INFO response
         info_resp = utils.ensure_str(self.connection.read_response())
 
-        version = re.findall(r"^redis_version:([\d\.]+)\r\n", info_resp, re.MULTILINE)[0]
+        version = re.findall(r"^redis_version:([\d\.]+)\r\n", info_resp, re.MULTILINE)[
+            0
```
Comment (guoweikuang) [inline iredis/client.py]: This place is not particularly well formatted.

File: .circleci/config.yml [hunk 2]
```diff
@@ -126,7 +126,7 @@ jobs:
           name: black format
           command: |
             . venv/bin/activate
-            black --check
+            black --check iredis/
```
Comment (laixintao) [inline .circleci/config.yml]: I think the tests/ also need tests. Why add `iredis/` ?

Comment (qiaocci) [reply .circleci/config.yml]: 如果直接运行`black --check .`，格式化项目的同时会格式化虚拟环境的第三方包，这不是我们想要的效果，参考（https://circleci.com/gh/laixintao/iredis/1937?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link）。
所以需要指定格式化的目录，iredis/, scripts/, tests/。

Comment (laixintao) [reply .circleci/config.yml]: 我明白了，没给路径的时候 black 退出状态是 0，导致之前的 Ci 一直没运行，但是我没发现。

image

File: iredis/client.py (hunk 1 above)
Comment (laixintao) [reply iredis/client.py]: Black's uncompromising... I can accept that.

**Comment by laixintao** [comment]
Thanks!

**Pull request merged**